src/app/components/layouts/tag/terra-tag.component.html

```diff
@@ -1,9 +1,13 @@
-<div class="tag" [ngClass]="inputCustomClass">
+<div class="tag"
+     [ngStyle]="{ 'background-color' : getBgColor() }"
+     [ngClass]="inputCustomClass">
     <span class="tag-icon"
           *ngIf="inputIsTaggable"
+          [ngStyle]="{ 'color': getColor() }"
           [ngClass]="{'isTagged': inputIsTagged, 'icon-ticket_prio1': inputIsTagged, 'icon-ticket_prio8': !inputIsTagged}">
     </span>
-    <span class="tag-text">
+    <span class="tag-text"
+          [ngStyle]="{ 'color': getColor() }">
```

Comment:
Leider komme ich erst jetzt zum Review, möchte aber trotzdem noch ein paar Anmerkungen anlässlich Wissensvermittlung da lassen: @estafilarakis @nappynapster @dantrau 
* Man sollte sich bewusst sein, dass wenn Funktionen im Template angegeben werden, diese bei jedem Change-Detection-Zyklus aufgerufen werden. Um ein Gefühl für die Häufigkeit zu bekommen, einfach mal nen Breakpoint in die Funktion `getColor()` setzen. Es sollte also **immer** der Zugriff auf eine Variable der Component statt der Aufruf einer Funktion bevorzugt werden!! Die Variable kann dann bei der Initialisierung gesetzt und kann bei Wertänderungen an Inputs in der Lifecycle-Hook-Methode `ngOnChanges` aktualisiert werden. Bei jedem Change-Detection-Zyklus wird so dann **nur der Wert** der Variablen ausgelesen.
* Methoden die im Template verwendet werden müssen entweder den access modifier `protected` oder `public` besitzen. Ein Zugriff auf `private` Funktionen der Component aus dem Template heraus ist in Angular zwar möglich aber sehr unschön. Weder die IDE noch der Compiler kann einen solchen Zugriff auflösen und daher die Verwendung der Funktion nicht identifizieren.
* Nur eine Geschmackssache: Statt einer Funktion mit dem Prefix _get_ kann ein getter angeboten werden. Dieser liest sich im Template dann auch wie eine Zugriff auf eine Variable. Bsp:
```
public get Color():string
{
     if(!isNullOrUndefined(this.inputColor))
     {
             return (new Color(this.inputColor)).isDark() ? '#ffffff' : '#000000';
     }
     return null;
}
```
Zugriff im Template:
`[ngStyle]={ 'color': Color }`
Die zuvor beschriebene Problematik besteht aber auch hier. Daher ist auch diese Variante eher selten einzusetzen.